build-aux/flatpak/io.github.lainsce.Notejot.json

```diff
@@ -6,11 +6,8 @@
     "command" : "io.github.lainsce.Notejot",
     "rename-icon" : "io.github.lainsce.NotejotDevel",
     "finish-args" : [
-        "--filesystem=xdg-documents",
-        "--filesystem=xdg-download",
```

Comment:
I removed this two as notes are store under ~/.var/app/io.github.lainsce.Notejot for now